using_images/sti_images/php.adoc

```diff
@@ -0,0 +1,82 @@
+= PHP Image
+{product-author}
+{product-version}
+:data-uri:
+:icons:
+:experimental:
+:toc: macro
+:toc-title:
+
+toc::[]
+
+== Overview
+OpenShift provides https://github.com/openshift/source-to-image[STI] enabled PHP images for building and running PHP
+applications.  These images can be used with the link:../../architecture/core_objects/builds.html#sti-build[STI build strategy].
+
+== Versions
+Currently, OpenShift only provides version https://github.com/openshift/sti-php/tree/master/5.5[5.5] of PHP.
+
+== Images
+
+The following base images are currently available:
+
+* RHEL 7
+* CentOS 7
+
+*RHEL 7 Base Image*
+
+The RHEL 7 image is available through RedHat's subscription registry via
+
+****
+`$ docker pull registry.access.redhat.com/openshift/php-55-rhel7`
+****
+
+*CentOS 7 Base Image*
+
+This image is available on DockerHub. To download it:
+
+****
+`$ docker pull openshift/php-55-centos7`
+****
+
+To use these images, you can either access them directly in those registries or push them into your OpenShift docker registry.  
+In addition, it is recommended that you create an ImageStream that points to the image, either in your docker registry
+or at the external location.  Your OpenShift resources can then reference the ImageStream.  You can find example ImageStream definitions for all the provided OpenShift images https://github.com/openshift/origin/examples/image-streams[here].
```

Comment:
This link also I assume should be: https://github.com/openshift/origin/tree/master/examples/image-streams ?